cli/wallet/wallet.go

```diff
@@ -229,6 +237,13 @@ func NewCommands() []cli.Command {
 				Action:    signStoredTransaction,
 				Flags:     signFlags,
 			},
+			{
+				Name:      "testinvoke",
```

Comment:
So it's `wallet testinvoke`, but it doesn't use wallet in any way. `util txdump` (and drop `--dump` and use positional `file.in`)?